pkg/paravirt/paravirt_protected_entity.go

```diff
@@ -46,7 +46,7 @@ func (this ParaVirtProtectedEntity) GetInfo(ctx context.Context) (astrolabe.Prot
 		}
 	}
 
-	return astrolabe.NewProtectedEntityInfo(this.id, name, this.data, this.metadata, this.combined, []astrolabe.ProtectedEntityID{}), nil
+	return astrolabe.NewProtectedEntityInfo(this.id, name, -1, this.data, this.metadata, this.combined, []astrolabe.ProtectedEntityID{}), nil
```

Comment:
Please think through what does -1 means here and why it is safe to use -1 here. Then add some comments above this line.

Reply:
Add one line comments.